app/build.gradle

```diff
@@ -46,4 +46,5 @@ dependencies {
     implementation 'com.squareup.retrofit2:retrofit:2.7.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
     implementation "android.arch.lifecycle:extensions:1.1.1"
+    implementation 'com.squareup.okhttp3:okhttp:4.9.0'
```

Comment:
This needed to be added, as I was getting "Expected Android API level 21+ but was 30" errors when I tried running the code. I found this solution at https://stackoverflow.com/questions/63917431/expected-android-api-level-21-but-was-30 if you want to read a bit more about the issue.

Reply:
Good.